framework/doc/content/documentation/systems/Mesh/Partitioner/PartitionerWeight.md

```diff
@@ -0,0 +1,7 @@
+# PartitionerWeight
+
+It is a test `partitioner` to demonstrate how to assign the vertex weights and the edge weights to the graph.
```

Comment:
Let's remove `vertex` from the documentation... that means "nodes" to many people.  Let's just talk about "element weights" and "side weights" for now.